.github/workflows/build-wheels.yml

```diff
@@ -0,0 +1,31 @@
+name: Build wheels
+
+on:
+  workflow_dispatch
+
+jobs:
+  buildwheels:
+    strategy:
+      matrix:
+        os: [ubuntu-latest, macos-latest]
+        python-version: [3.6]
+
+    runs-on: ${{ matrix.os }}
+
+    steps:
+    - name: install automake
+      run: brew install automake
+      if: matrix.os == 'macOS-latest'
+    - uses: actions/checkout@v2
+    - uses: actions/setup-python@v2
+    - name: install cibuildwheel
+      run: python -m pip install cibuildwheel==2.2.0a1
+    - name: Build wheels
+      run: python -m cibuildwheel --output-dir wheelhouse
+      env:
+        CIBW_BUILD_VERBOSITY: 1
+        CIBW_BEFORE_BUILD: "./ci/build-parasail.sh"
+        CIBW_SKIP: "cp36-* cp310-macosx_* pp3*"
```

Comment:
any specific reason why we're skipping `cp36-*` and `cp310-macosx_*`?

EDIT : I'm confused now. The matrix only has Python 3.6 but we're skipping `cp36-*` in the build wheels?

Reply:
Thanks for that - I knew I was missing something trivial.

Reply:
I've added an issue for the Python 3.10 macOS wheel: https://github.com/enthought/vpsearch/issues/12